plugins/woocommerce/src/Gateways/PayPal/AddressRequirements.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Gateways\PayPal;
+
+/**
+ * Class AddressRequirements
+ *
+ * This helper class checks country-specific address requirements.
+ * This was built based on https://developer.paypal.com/api/rest/reference/orders/v2/country-address-requirements/
```

Comment:
Nit: It may be worth adding an explicit `@see` annotation.
```suggestion
 * This was built based on {@see https://developer.paypal.com/api/rest/reference/orders/v2/country-address-requirements/}
```